cmd/terramate/e2etests/cloud/interop/interoperability_test.go

```diff
@@ -16,14 +16,35 @@ import (
 )
 
 func TestInteropCloudSyncPreview(t *testing.T) {
```

Comment:
if this test can only run in GHA, please check for the `CI` env var and use `t.Skip()` if not set.